documentation.md

```diff
@@ -157,7 +157,6 @@ A Riot component is a combination of layout (HTML) and logic (JavaScript). Here
 * The `this` keyword is optional: `<p>{ name }</p>` and `<p>{ this.name }</p>` are both valid
 * Quotes are optional: `<foo bar={ baz }>` and `<foo bar="{ baz }">` are both valid
 * Boolean attributes (checked, selected, etc.) are ignored when the expression value is falsy: `<input checked={ undefined }>` becomes `<input>`.
-* All attribute names *must be lowercase*. This is due to browser specification.
```

Comment:
Thats great. After Riot3 I did happy find out what passed props can be in camel because it is not used in final DOM and I use it in the same way in my project:

child.riot
```
<child>{ props.someCoolProperty }</child>
```

app.riot
```
<app>
<child someCoolProperty={ state.foo }></child>
</app>
```

Is it pretty easy for understand and much more logical than:
```
<app>
<child some-cool-property={ state.foo }></child>
</app>
```

I would like to add some tests to dom-bindings to check this behavior won't broken after some updates.

Reply:
> I would like to add some tests to dom-bindings to check this behavior won't broken after some updates.

I thought they are already in place, if not go for it ;)